config/webpack.config.prod.js

```diff
@@ -1,15 +1,16 @@
 const webpackCommon = require('./webpack.common.config');
+const path = require('path');
 
 const config =  {
   entry: {
-    'react-data-grid/dist/react-data-grid': ['./packages/react-data-grid/src'],
-    'react-data-grid-addons/dist/react-data-grid-addons': ['./packages/react-data-grid-addons/src'],
-    'react-data-grid/dist/react-data-grid.min': ['./packages/react-data-grid/src'],
-    'react-data-grid-addons/dist/react-data-grid-addons.min': ['./packages/react-data-grid-addons/src'],
-    'react-data-grid-examples/dist/index': './packages/react-data-grid-examples/src'
+    'react-data-grid/dist/react-data-grid': [path.resolve('packages/react-data-grid/src')],
```

Comment:
It seems like both `react-data-grid` and `react-data-grid.min.js` are minified. 

![image](https://user-images.githubusercontent.com/1452717/48557224-d0818180-e8ab-11e8-9402-419a377a8275.png)